src/meta-srv/src/table_meta_alloc.rs

```diff
@@ -45,15 +45,14 @@ impl MetasrvPeerAllocator {
     /// table or not. So this method isn't idempotent.
     async fn alloc(
         &self,
-        ctx: &TableMetadataAllocatorContext,
+        _ctx: &TableMetadataAllocatorContext,
```

Comment:
```suggestion
```

Reply:
Remove `TableMetadataAllocatorContext` completely in b57fa8096ff20e06b42eb351cb538893a64ceb7a